08-confidence-intervals.Rmd

```diff
@@ -26,21 +26,39 @@ options(pillar.sigfig = 6)
 set.seed(76)
 ```
 
-In Chapter \@ref(sampling), we studied sampling. We started with a "tactile" exercise where we wanted to know the proportion of balls in the sampling bowl in Figure \@ref(fig:sampling-exercise-1) that are red. While we could have performed an exhaustive count, this would have been a tedious process. So instead, we used a shovel to extract a sample of 50 balls and used the resulting proportion that were red as an *estimate*. Furthermore, we made sure to mix the bowl's contents before every use of the shovel. Because of the randomness created by the mixing, different uses of the shovel yielded different proportions red and hence different estimates of the proportion of the bowl's balls that are red. 
+```{r echo=FALSE, purl=FALSE}
+# This code is used for dynamic non-static in-line text output purposes
+p_red <- 0.375
```

Comment:
Just to be very cautious here how about `p_red <- moderndive::bowl %>% dplyr::summarize(mean(color == "red")) %>% pull()` instead?